include/libp2p/crypto/rsa_types.hpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef LIBP2P_CRYPTO_RSA_TYPES_HPP
+#define LIBP2P_CRYPTO_RSA_TYPES_HPP
+
+#include <vector>
+
+namespace libp2p::crypto::rsa {
+  /**
+   * @brief Common types
+   */
+  using Key = std::vector<uint8_t>;       /**< Common type for keys */
```

Comment:
Is key in rsa fixed size or could be arbitrary size? If fixed please update to array. Same for signature one line below

Reply:
If size of private and public keys are different consider to make different types for them